checkly/resource_alert_channel.go

```diff
@@ -161,14 +166,35 @@ func resourceAlertChannel() *schema.Resource {
 					},
 				},
 			},
+			AcFieldSendRecovery: {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			AcFieldSendFailure: {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			AcFieldSendDegraded: {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			AcFieldSSLExpiry: {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			AcFieldSSLExpiryThreshold: {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  30,
```

Comment:
How does Optional and Default work together? Are we sending 30 if it is not set?

Reply:
yea, unnecessary, removing!